### PR TITLE
Translated and updated README

### PR DESCRIPTION
**Pull request by Guilhermeasper**

### CODE REVIEW

File: README_en-US.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
+# Rusbé
+
+![GitHub package.json version](https://img.shields.io/github/package-json/v/erick2280/rusbe-web-client)
+
+Rusbé is a follow up platform of the University Restaurant of UFPE.
+
+## Setup
+
+### Configurando
```
Comment (MarconiGRF) [inline README_en-US.md]: I think this line "Configurando" should be removed.

Comment (Guilhermeasper) [reply README_en-US.md]: Yes, my mistake, it has already been fixed.

File: README_en-US.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
+# Rusbé
+
+![GitHub package.json version](https://img.shields.io/github/package-json/v/erick2280/rusbe-web-client)
+
+Rusbé is a follow up platform of the University Restaurant of UFPE.
+
+## Setup
+
+### Configuring
```
Comment (MarconiGRF) [inline README_en-US.md]: Setup and configuring communicates pretty much the same idea, so I would kepe only "Setup" line.

Comment (Guilhermeasper) [reply README_en-US.md]: I've merged the two sections into only one called "Setting up".

**Comment by MarconiGRF** [comment]
Looks ok to me now :)

**Pull request merged**